src/test_workflow/integ_test/integ_test_suite_opensearch.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import glob
+import logging
+import os
+
+from test_workflow.integ_test.integ_test_suite import IntegTestSuite, InvalidTestConfigError
+from test_workflow.integ_test.local_test_cluster import LocalTestCluster
+from test_workflow.test_result.test_component_results import TestComponentResults
+from test_workflow.test_result.test_result import TestResult
+
+
+class IntegTestSuiteOpenSearch(IntegTestSuite):
+    """
+    Kicks of integration tests for a component based on test configurations provided in
+    test_support_matrix.yml
+    """
+
+    def __init__(
+        self,
+        dependency_installer_opensearch,
+        component,
+        test_config,
+        bundle_manifest_opensearch,
+        build_manifest_opensearch,
+        work_dir,
+        test_recorder
+    ):
+        super().__init__(
+            work_dir,
+            component,
+            test_config,
+            test_recorder,
+            dependency_installer_opensearch,
+            bundle_manifest_opensearch,
+            build_manifest_opensearch
+        )
+
+    def execute_tests(self):
```

Comment:
TestSuite.execute is more concise than TestSuite.execute_tests, unless there are other executions in this suite. For my learning, is there a specific reason to change the function name?

Reply:
It also imports system execute. https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/integ_test/integ_test_suite.py#L14 To avoid confusion, I call it execute_tests